TechnologyEnhancedLearning/DLSV2
# HEEDLS-388 - Moved UserDataService to DataService folder and implemen…

…ted inactive centre handling on log in

Tested:
- Ran all unit tests
- Manually tested in Chrome, IE11, Edge, Firefox
- Chrome mobile view
- No JS
- Screen reader

I'm wondering if the paragraph on the new CentreInactive page (screenshot below) should be split into two paragraphs (one for each sentence)? UPDATE: after the Show and Tell on Friday 7th of May, it was decided to keep the page as it currently is - so there is no need for any restyling or new content.

New CentreInactive error page:

![Centre Inactive error page](https://user-images.githubusercontent.com/80777689/117294190-460dad80-ae6a-11eb-8297-071c5bf24e3c.PNG)
